base/bin/migration.mustache

```diff
@@ -2,9 +2,9 @@ const fs = require('fs');
 const mustache = require('mustache');
 
 const now = new Date().toISOString();
-const migrationName = now + '_' + process.argv[2];
+const migrationName = `${now}_${process.argv[2]}`;
```

Comment:
Opened this box to say "Don't these mandate node 6?". Figured I should check and they're sure valid in node 4! Why have I been concatenating strings like an idiot all this time?